org.gnome.Glade.json

```diff
@@ -30,35 +27,13 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/glade/3.22/glade-3.22.2.tar.xz",
-                    "sha256": "edefa6eb24b4d15bd52589121dc109bc08c286157c41288deb74dd9cc3f26a21"
-                },
-                {
-                    "type": "patch",
-                    "path": "appdata.patch"
+                    "url": "https://download.gnome.org/sources/glade/3.36/glade-3.36.0.tar.xz",
+                    "sha256": "19b546b527cc46213ccfc8022d49ec57e618fe2caa9aa51db2d2862233ea6f08"
                 }
             ],
             "modules": [
                 "shared-modules/intltool/intltool-0.51.json"
             ]
-        },
```

Comment:
I don't think removing libhandy is a great idea as we will lose the glade catalogue of it

Reply:
Sure, we need @Kekun to do a new release as 0.13 do not compile with new glade version. I think it is solved on libhandy master.

One option is to add an extension point to allow to include catalogs on demand, instead of bundle them on glade.

Reply:
Ah right, feel free to merge then